library/src/main/java/org/pattonvillerobotics/commoncode/robotclasses/drive/MecanumEncoderDrive.java

```diff
@@ -144,10 +151,11 @@ public void moveInches(Direction direction, double inches, double power) {
         Telemetry.Item distance = telemetry("DistanceL: N/A DistanceR: N/A");
         Telemetry.Item distanceRear = telemetry("DistanceLR: N/A DistanceRR: N/A");
 
-        while (isMovingToPosition() || !motorsReachedTarget(targetPositionLeft, targetPositionRight, targetPositionLeftRear, targetPositionRightRear) && !linearOpMode.isStopRequested() && linearOpMode.opModeIsActive()) {
-            Thread.yield();
```

Comment:
@greg-bahr Explanation: `Thread.yield()` is already called in `linearOpMode.opModeIsActive()`